src/main/java/mate/jdbc/dao/CarDaoImpl.java

```diff
@@ -0,0 +1,153 @@
+package mate.jdbc.dao;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.exception.DataProcessingException;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.util.ConnectionUtil;
+
+@Dao
+public class CarDaoImpl implements CarDao {
+    @Override
+    public Car create(Car car) {
+        String query = "INSERT INTO cars (model, manufacturer_id) VALUES(?, ?)";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement createStatement =
+                        connection.prepareStatement(query, Statement.RETURN_GENERATED_KEYS)) {
+            createStatement.setString(1, car.getModel());
+            createStatement.setLong(2, car.getManufacturer().getId());
+            createStatement.executeUpdate();
+            ResultSet result = createStatement.getGeneratedKeys();
+            if (result.next()) {
+                car.setId(result.getObject(1, Long.class));
+            }
+            return car;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Couldn't create car " + car + ". ", e);
+        }
+    }
+
+    @Override
+    public Optional<Car> get(Long id) {
+        String query = "SELECT C.id, C.model, "
+                + "M.id, M.name, M.country, "
+                + "D.id, D.name, D.license_number "
+                + "FROM cars C "
+                + "INNER JOIN manufacturers M ON C.manufacturer_id = M.id "
+                + "INNER JOIN cars_drivers CD ON C.id = CD.car_id "
+                + "INNER JOIN drivers D ON CD.driver_id = D.id "
```

Comment:
```suggestion
                + "INNER JOIN manufacturers m ON C.manufacturer_id = M.id "
                + "INNER JOIN cars_drivers cd ON C.id = CD.car_id "
                + "INNER JOIN drivers d ON CD.driver_id = D.id "
```
always use lowercase for sql aliases